Code/skyrim_ui/src/app/components/settings/settings.component.ts

```diff
@@ -1,25 +1,24 @@
-import { Component, EventEmitter, HostListener, OnInit, Output } from '@angular/core';
+import { Component, EventEmitter, HostListener, Output } from '@angular/core';
 import { TranslocoService } from '@ngneat/transloco';
 import { ClientService } from 'src/app/services/client.service';
 import { FontSize, SettingService } from 'src/app/services/setting.service';
 import { Sound, SoundService } from '../../services/sound.service';
 
 export enum PartyAnchor {
-  TOP_LEFT,
-  TOP_RIGHT,
-  BOTTOM_RIGHT,
-  BOTTOM_LEFT,
+  TOP_LEFT = 'top_left',
+  TOP_RIGHT = 'top_right',
+  BOTTOM_RIGHT = 'bottom_right',
+  BOTTOM_LEFT = 'bottom_left',
 }
 
+export const autoHideTimerLengths = [1, 3, 5].map(l => l.toFixed(0));
```

Comment:
I think that this should be an array of numbers, not strings and that the SelectSettings' options parameter should be an `any[]` or a `(string | number)[]`